tests/modules/nf-core/vgan/haplocart/main.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { VGAN_HAPLOCART } from '../../../../../modules/nf-core/vgan/haplocart/main.nf' addParams( options: [args: ''] )
+
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.gg", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.og", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.xg", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.giraffe.gbz", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.dist", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph_paths", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/path_supports", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/graph.gbwt", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/children.txt", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/parents.txt", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/parsed_pangenome_mapping", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/mappability.tsv", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/k17_w18.min", checkIfExists: true)
+    file("ftp://ftp.healthtech.dtu.dk/public/haplocart/hcfiles/k31_w11.min", checkIfExists: true)
```

Comment:
You should use a minimalist dataset based in `nf-core/test-datasets` in branch `modules` as test data. Is it possible to move these files there? (and possible make them smaller?)

Reply:
Sure I can move them there! But I do not think I can make them any smaller, the large ones are already compressed.